src/main/steps/applicant1/hub-page/right-column/content.ts

```diff
@@ -67,6 +67,7 @@ export const generateContent: TranslationFn = content => {
   const aosSubmitted =
     !content.isJointApplication &&
     (content.userCase.applicant2StatementOfTruth ||
+      content.userCase.aosStatementOfTruth ||
```

Comment:
same as below, because we're checking that it's a sole application we don't need to check for applicant2StatementOfTruth

Reply:
See NFDIV-2297